JSQMessagesDemo/DemoMessagesViewController.m

```diff
@@ -111,7 +115,17 @@ - (void)viewDidAppear:(BOOL)animated
     self.collectionView.collectionViewLayout.springinessEnabled = [NSUserDefaults springinessSetting];
 }
 
+- (void)preferredContentSizeChanged:(NSNotification *)notification
+{
+    [self.collectionView layoutSubviews];
```

Comment:
I am assuming that we want to invalidate the `CollectionViewLayout` then call `setNeedsLayout`. Just because that makes more sense to me. Is that correct?